### DISCUSSION TITLE
How to boot turtle's startup.lua without rightclicking it

### DISCUSSION THREAD

**Discussion by Elphyn** [cc-tweaked/CC-Tweaked#1794] [Q&A] [answered]
I was building a simple quarry, [code](https://github.com/Elphyn/miner), the problem is when turtle spawns another(on top of diskdrive with a disk/startup) it runs this startup only when I right click this turtle
I've seen Michael Reeves did quarry where his turtles got to work without any intervention on his part, they were just spawned and went to work, how can I do the same?
Mod: cc-tweaked-1.20.1-forge-1.110.1
I hope there's something, appreciate any help in regards to this question.

**Comment by SquidDev** [comment, marked as answer]
You can use the turtle as a peripheral, and then [turn it on](https://tweaked.cc/peripheral/computer.html#v:turnOn). So if you've got a turtle to the left of a computer, something like `peripheral.call("left", "turnOn")` should work.

**Comment by Elphyn** [comment]
Thank you!
